docs/tutorials/aws.md

```diff
@@ -84,7 +84,7 @@ spec:
         args:
         - --source=service
         - --source=ingress
-        - --domain-filter=external-dns-test.my-org.com. # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
+        - --domain-filter=external-dns-test.my-org.com # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
```

Comment:
@jrnt30 no, but would be interesting to hear if you don't mind explaining the process in the docs (maybe create a PR with modified contribution guidelines) ?